### PR TITLE
1774 Add state param to webSessionUrl

### PR DESCRIPTION
**Pull request by astrimagnussen**

### CODE REVIEW

File: webflows/src/main/java/com/schibsted/account/webflows/api/SchibstedAccountApi.kt [hunk 1]
```diff
@@ -102,12 +102,22 @@ internal class SchibstedAccountApi(baseUrl: HttpUrl, okHttpClient: OkHttpClient)
 
     fun sessionExchange(
         user: User,
+        state: String?,
```
Comment (saltwoo) [inline webflows/src/main/java/com/schibsted/account/webflows/api/SchibstedAccountApi.kt:105]: maybe add `state` after redirectUri ?

File: webflows/src/main/java/com/schibsted/account/webflows/user/User.kt [hunk 2]
```diff
@@ -89,13 +89,24 @@ class User {
     /**
      * Generate URL with embedded one-time code for creating a web session for the current user.
      *
+     * @param state optional, used for adding a path to the redirectUri where user needs to be redirected in order to be authenticated
```
Comment (saltwoo) [inline webflows/src/main/java/com/schibsted/account/webflows/user/User.kt:92]: I believe this description will confuse people so let's use the same one we use at our [browser SDK](https://github.com/schibsted/account-sdk-browser/blob/master/src/identity.js#L21)

**Review by saltwoo — CHANGES_REQUESTED** [review]
Overall looks good ! Just added 2 comments

File: webflows/src/main/java/com/schibsted/account/webflows/api/SchibstedAccountApi.kt (hunk 1 above)
Comment (astrimagnussen) [reply webflows/src/main/java/com/schibsted/account/webflows/api/SchibstedAccountApi.kt:105]: sure 👍

**Review by saltwoo — APPROVED** [review]
LGTM 👍

**Pull request merged**